tools/scoring/score/dimensions/standard_field_naming.py

```diff
@@ -23,33 +22,41 @@
 
 
 class StandardFieldNaming(Dimension):
-  """
-  Quantifies whether the correct standard field names
+  """Quantifies whether the correct standard field names
   (e.g. "chilled_water_flowrate_sensor")
-  were selected in the proposed file.
-  """
-  def __init__(self, *, translations: TranslationsDict):
-    super().__init__(translations=translations)
+  were selected in the proposed file."""
+  def _split_subfields(self, field):
+    return set(
+        filter(lambda subfield: not bool(regex.match('[0-9]+', subfield)),
+               field.split('_')))
+
+  def evaluate(self):
+    """Calculates and assigns properties necessary for generating a score."""
+
+    proposed_condensed, solution_condensed = map(self._condense_translations,
+                                                 (PROPOSED, SOLUTION))
+
+    # Account for empty list
+    proposed_translations = proposed_condensed and proposed_condensed[0]
+    solution_translations = solution_condensed and solution_condensed[0]
 
     correct_subfields = []
     correct_ceiling: int = 0
     incorrect_subfields = []
 
-    for s_field, s_value in translations[SOLUTION]:
-      s_subfields = set(
-          filter(lambda subfield: not bool(regex.match('[0-9]+', subfield)),
-                 s_field.split('_')))
+    for s_field, s_value in solution_translations:
```

Comment:
nit: could you rename "s_field" and "s_value" to something like "solution_field" and "solution_value"? Same comment for p_field and p_value below

Reply:
Good suggestion. https://github.com/google/digitalbuildings/pull/499/commits/b2073f74e5f0bb2c35a7e34a578488bafeb49222